ArkhamDisplay/Data.cs

```diff
@@ -186,7 +186,10 @@ public static Theme UseTheme()
                 return registryValue > 0 ? Theme.Light : Theme.Dark;
             }
         }
-        catch (Exception) { }
+        catch (Exception ex)
```

Comment:
This was intentional. A registry call failing is not strictly an error, and I would rather not bother the user in this case since it's handled gracefully (i.e. default to dark mode).

I suppose we should catch the kinds of exceptions we're expecting first, handle those gracefully, and if we get an unexpected one then show an error message. I'm not overly concerned about it in this particular context though.